Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs

```diff
@@ -213,7 +213,8 @@ public static void ParseBuildCommandLine(ref IBuildInfo buildInfo)
                         buildInfo.AutoIncrement = true;
                         break;
                     case "-scenes":
-                        // TODO parse json scene list and set them.
+                        buildInfo.Scenes = from scene in arguments[++i].Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
@StephenHodgson you mentioned concerns about this linq statement. You're not convinced it covers all cases? Perhaps you could elaborate on some of you concerns.

Reply:
Where was that comment?

I think my main concern was if the paths defined were relative or absolute.

Reply:
In my testing of this I had to define my paths relative to the project root or Unity would complain at me.

Reply:
My last question. If I had a json file that defined my scene list, could I use that instead?

Seems a bit easier than writing it all in the command line arg.

Reply:
Both absolute paths and paths relative to the project root will work here.

The command line list may be a touch easier on CI as we'd either have to create a file on the fly or check one into the repo. Maybe we could have both? 
-sceneList: comma separated list string provided directly to the command line
-sceneListJson: path to json file that contains a scene list

If both are provided we take the union.

Reply:
I think providing an scene list file is probably the best way to go here, but does this change add that functionality or is it just a csv string?

Reply:
Because I also had plans of creating a helper menu in the build window to create a scene list that would then spit out a json file at a known location (likely the root of the build folder)

Reply:
I do think that we should allow for a csv string as well. I'm happy to have a scene list file option too, but for automated builds, the csv string is one step less. 
Is my proposal for two command line switches above not acceptable?

Reply:
```
case "-sceneList":
    buildInfo.Scenes.Union(from scene in arguments[++i].Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
                           select scene.Trim());
    break;
case "-sceneListJson":
    string path = arguments[i++];
    if (File.Exists(path))
    {
        buildInfo.Scenes.Union(JsonUtility.FromJson<List<string>>(File.ReadAllText(path)));
    }
    break;
```
Something like this?

Reply:
Looks great!

Reply:
So I just tested this on Device, and unfortunately it hadn't built the scenes in the json file. I ended up finding some warnings in the build output. Turns out Unity JsonUtility does support deserializing top level json lists ([link](https://forum.unity.com/threads/how-to-load-an-array-with-jsonutility.375735/#post-2437941)). We'd need to wrap this in an object, and then deserialize to an c# object that contains a string list.

So rather than complicate and confuse things, I'm just going to allow for reading csv from file as well as from command line.